src/statistics.jl

```diff
@@ -560,8 +560,8 @@ var(M::Manifold, x::AbstractVector; kwargs...) = mean_and_var(M, x; kwargs...)[2
 compute the optionally weighted standard deviation of a `Vector` `x` of `n` data
 points on the [`Manifold`](@ref) `M`, i.e.
 
-````math
-\sqrt{\frac{1}{c} \sum_{i=1}^n w_i d_{\mathcal M}^2 (x_i,m)},
+````math𝒩
```

Comment:
This is probably not correct.

Reply:
Thanks, I missed that one. Will correct that.